ci/tests/rp-parsl/test.sh

```diff
@@ -14,4 +14,4 @@ python map_reduce.py
 ret=$?
 echo "--- smoke test $ret"
 
-exit $ret
+echo "Success!"
```

Comment:
Same here btw - we do want the exit code to propagate...

Reply:
Ok, thought to keep in sync with other tests - in a way to collect return code (status of tests running), but not to set a corresponding code for the script itself (thus, to distinguish failed script and failed tests)